src/database/migrations/z1640788383-populate-airgrab.js

```diff
@@ -0,0 +1,114 @@
+'use strict';
+
+const fs = require('fs');
+const fetch = require('node-fetch');
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    const url = 'https://storageapi.fleek.co/obernardovieira-26385-team-bucket/merkleTree.json';
```

Comment:
prevent from running on tests or non-profuction envs
```suggestion
    if (process.env.NODE_ENV === 'test' || process.env.API_ENVIRONMENT !== 'production') {
        return;
    }
    const url = 'https://storageapi.fleek.co/obernardovieira-26385-team-bucket/merkleTree.json';
```